src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -104,7 +104,7 @@ public function getEntitiesByIds($identifier, array $values)
             // Filter out non-integer values (e.g. ""). If we don't, some
             // databases such as PostgreSQL fail.
             $values = array_values(array_filter($values, function ($v) {
-                return (string) $v === (string) (int) $v;
+                return (string) $v === (string) (int) $v || ctype_digit($v);
```

Comment:
Why aren't you only using `ctype_digit()`? Does the first test anything, that the second test does not?